src/main/java/cam72cam/immersiverailroading/library/PaintBrushMode.java

```diff
@@ -0,0 +1,6 @@
+package cam72cam.immersiverailroading.library;
+
+public enum PaintBrushMode {
+	SEQUENTIAL,
```

Comment:
Do we want to translate these as well?  I'm not super consistent about it through the rest of the code, but it probably makes sense here

Reply:
I think you can, see how the coupler enum works

Reply:
I see!